extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,107 @@
-  and ask what to do.
+    <span class="heading">No private email</span>.
+    Do not send comments to [% term.bugs %] by private email; no-one else can read
+    them if you do that, and they'll be missed and/or ignored. If an attachment
+    is too big for [% terms.Bugzilla %], add a comment giving the file size and contents
+    and ask what to do.
   </li>
 </ol>
 
-<h3>2. Changing Fields</h3>
+<h3>Changing Fields</h3>
 
 <ol>
   <li>
-  <span class="heading">No messing with other people's [% terms.bugs %]</span>.
-  Unless you are the [% terms.bug %] assignee, or have some say over the use of their
-  time, never change the Priority or Target Milestone fields. If in doubt,
-  do not change the fields of [% terms.bugs %] you do not own - add a comment
-  instead, suggesting the change.
+    <span class="heading">No messing with other people's [% terms.bugs %]</span>.
+    Unless you are the [% terms.bug %] assignee, or have some say over the use of their
+    time, never change the Priority or Target Milestone fields. If in doubt,
+    do not change the fields of [% terms.bugs %] you do not own - add a comment
+    instead, suggesting the change.
   </li>
 
   <li>
-  <span class="heading">No whining about decisions</span>.
-  If a respected project contributor has marked a [% terms.bug %] as INVALID, then it is
-  invalid. Someone filing another duplicate of it does not change this. Unless
-  you have further important evidence, do not post a comment arguing that an
-  INVALID or WONTFIX [% terms.bug %] should be reopened.
-  </li>
-
-</ol>
+    <span class="heading">No whining about decisions</span>.
+    If another project contributor has marked a [% terms.bug %] as INVALID, then it is
+    invalid. Filing another duplicate of it does not change this. Unless
+    you have further evidence to support this, do not post a comment arguing that an
+    INVALID or WONTFIX [% terms.bug %] should be reopened. 
+  </li> 
 
-<h3>3. Applicability</h3>
-
-<ol>
-  <li>
-  Some of these rules may not apply to you. If they do not, you will know
-  exactly which ones do not, and why they do not apply. If you are not
-  sure, then they definitely all apply to you.
-  </li>
 </ol>
 
 <p>
-  If you see someone not following these rules, the first step is, as an exception
-  to guideline 1.4, to make them aware of this document by <em>private</em> mail.
-  Flaming people publically in [% terms.bugs %] violates guidelines 1.1 and 1.3. In the case of
-  persistent offending you should ping an administrator on Mozilla IRC in channel #bmo and ask them
-  to look into it.
-</p>
-
-<p>
-  This entire document can be summed up in one sentence:
-  do unto others as you would have them do unto you.
+  If you observe, or are the subject of behavior in violation of these guidelines, please tag
+  the [% terms.bug %] or comment following our <a href="page.cgi?id=anti-abuse.html">anti-abuse policy</a>.
```

Comment:
No, I should put in guidance here to tag comment #0 if the whole bug needs to be tagged.